bindings/cpp/logger.cpp

```diff
@@ -13,11 +13,17 @@
 #include <blackhole/record.hpp>
 #include <blackhole/logger.hpp>
 
+#include "example/config.hpp"
 #include "elliptics/session.hpp"
 #include "library/elliptics.h"
 
 namespace ioremap { namespace elliptics {
 struct trace {
+	struct trace_stack_t {
```

Comment:
Both `std::vector` and `std::deque` will allocate memory as needed, but `std::vector` will also copy stored data.